rootfs/etc/profile.d/_workdir.sh

```diff
@@ -0,0 +1,60 @@
+# Files in the profile.d directory are executed by the lexicographical order of their file names.
+# This file sets the working directory inside Geodesic to match the host directory Geodesic
+# was launched from, if possible. If the host directory is not accessible, it sets the working directory to `/`.
+#
+# This file is named _workdir.sh. The leading underscore is needed to ensure this file executes before
+# other files that may depend on it. The "w" is needed to ensure it is loaded *after* _preferences.sh
+#
+
+function _file_device() {
+	df --output=source "$1" | tail -1
```

Comment:
nitpick: some functions are spaced with a single space and other functions have double space